src/script/ml.ts

```diff
@@ -105,20 +107,31 @@ export async function trainModel(): Promise<tf.LayersModel | void> {
   const tensorFeatures = tf.tensor(features);
   const tensorLabels = tf.tensor(labels);
   const nn: LayersModel = createModel();
-  const totalNumEpochs = get(settings).numEpochs;
+  const totalNumEpochs = 160;
```

Comment:
Hardcode value here changing the initial ml settings won't affect anyone who has used the app before. Settings in local storage are preferentially used.